apps/taxes/src/pages/api/webhooks/order-created.ts

```diff
@@ -0,0 +1,105 @@
+import { SaleorAsyncWebhook } from "@saleor/app-sdk/handlers/next";
+import {
+  OrderCreatedEventSubscriptionFragment,
+  OrderStatus,
+  UntypedOrderCreatedSubscriptionDocument,
+  UpdateMetadataDocument,
+  UpdateMetadataMutation,
+  UpdateMetadataMutationVariables,
+} from "../../../../generated/graphql";
+import { saleorApp } from "../../../../saleor-app";
+import { createLogger } from "../../../lib/logger";
+import { getActiveTaxProvider } from "../../../modules/taxes/active-tax-provider";
+import { createClient } from "../../../lib/graphql";
+import { Client } from "urql";
+
+export const config = {
+  api: {
+    bodyParser: false,
+  },
+};
+
+type OrderCreatedPayload = Extract<
+  OrderCreatedEventSubscriptionFragment,
+  { __typename: "OrderCreated" }
+>;
+
+export const orderCreatedAsyncWebhook = new SaleorAsyncWebhook<OrderCreatedPayload>({
+  name: "OrderCreated",
+  apl: saleorApp.apl,
+  event: "ORDER_CREATED",
+  query: UntypedOrderCreatedSubscriptionDocument,
+  webhookPath: "/api/webhooks/order-created",
+});
+
+export const EXTERNAL_ID_KEY = "externalId";
```

Comment:
same for exporting symbols without context

Reply:
renamed and added comments

Reply:
renamed and added comments